charts/trino/values.yaml

```diff
@@ -1133,3 +1139,29 @@ networkPolicy:
   #        port: 9999
   # ```
   egress: []
+
+catalogsSecrets: []
```

Comment:
These values contain property files, not secrets - maybe we can choose a better name for them, without the word `secrets`? It's an implementation detail how we store contents of it and make it available in the deployments.

Reply:
Can you add a test for this somewhere in https://github.com/trinodb/charts/tree/main/tests/trino ?

Reply:
You have not addressed my comment here about the property name. Please choose a better name than `catalogsSecrets`.